pyproject.toml

```
@@ -0,0 +1,29 @@
+[build-system]
+requires = ["setuptools>=61.0.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "mypy_extensions"
+version = "1.0.0-dev"
+description = "Type system extensions for programs checked with the mypy type checker."
+readme = "README.md"
+license = {file = "LICENSE"}
```

```suggestion
license = {text = "MIT"}
```
I'd suggest to use the SPDX license string here instead. The license file is already included by setuptools.